CI.rst

```diff
@@ -59,7 +59,7 @@ Container Registry used as cache
 We are using GitHub Container Registry to store the results of the ``Build Images``
 workflow which is used in the ``Tests`` workflow.
 
-Currently in main version of Airflow we run tests in 4 different versions of Python (3.7, 3.8, 3.9, 3.10)
+Currently in main version of Airflow we run tests in those versions of Python (3.7, 3.8, 3.9, 3.10, 3.11)
 which means that we have to build 8 images (4 CI ones and 4 PROD ones). Yet we run around 12 jobs
```

Comment:
do we need to update the numbers here, now? yes, perhaps we need to update it back to the same numbers when we remove 3.7, but since we still have 3.7 in this PR 🤔 
Or maybe generalise this statement like we did with `those`, "that we have to build 2x images (one for CI and one for Prod for each of those versions)"

Reply:
Yeah. Why not, it's just a description :).